bin/cron/admin_progress

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+#
+# This script caches database queries whose results are displayed at studio.code.org/admin/progress.
+#
+require_relative('../../dashboard/config/environment')
+require 'cdo/properties'
+
+def main()
+  return unless only_one_running?(__FILE__)
+
+  SeamlessDatabasePool.use_persistent_read_connection do
+    user_count = User.count
+
+    levels_all = User.joins(:user_levels).group(:level_id)
+    levels_attempted = levels_all.where('best_result > 0').count
+    levels_passed = levels_all.where('best_result >= 20').count
```

Comment:
We might want to consider writing some [scopes](http://guides.rubyonrails.org/active_record_querying.html#scopes) for these various metrics (passed, perfect, etc) that we're starting to use consistently and that are based on an external understanding of our constants.